extension-objects/src/main/java/App.java

```diff
@@ -0,0 +1,60 @@
+import abstractextensions.CommanderExtension;
+import abstractextensions.SergeantExtension;
+import abstractextensions.SoldierExtension;
+import units.CommanderUnit;
+import units.SergeantUnit;
+import units.SoldierUnit;
+import units.Unit;
+
+/**
+ * Created by Srdjan on 26-Apr-17.
+ */
+public class App {
+
+  /**
+   * Program entry point
+   *
+   * @param args command line args
+   */
+  public static void main(String[] args) {
+
+    //Create 3 different units
+    Unit unit = new SoldierUnit("SoldierUnit1");
+    Unit unit1 = new SergeantUnit("SergeantUnit1");
+    Unit unit2 = new CommanderUnit("CommanderUnit1");
```

Comment:
Instead of unit, unit1 and unit2 how about soldierUnit, sergeantUnit and commanderUnit?